frontend/src/App.js

```diff
@@ -18,23 +24,7 @@ class App extends Component {
     return (
       <div className="home">
```

Comment:
btw why is the "home" className in the 'App' instead of the 'Home' component?

Reply:
I didn't notice this, sorry :( - replaced the div with React fragment